src/Utilities/SingleStackUtils/SingleStackUtils.jl

```diff
@@ -204,4 +207,58 @@ function get_horizontal_variance(
     return vars_var
 end
 
+"""
+    reduce_element_stack(
+        f::Function,
+        grid::DiscontinuousSpectralElementGrid{T, dim, N},
+        Q::MPIStateArray,
+        vars::NamedTuple,
+        var::String;
+        vrange::UnitRange = 1:size(Q, 3),
+    ) where {T, dim, N}
+
+Reduce `var` from `vars` within `Q` over all nodal points in the specified
+`vrange` of elements with `op`. Return a tuple `(result, z)` where `result` is
+the final value returned by `op` and `z` is the index within `vrange` where the
+`result` was determined.
+"""
+function reduce_element_stack(
```

Comment:
I think we need this to be `reduce_nodal_stack`, we can probably write two methods and use the same format as `get_vars_from_nodal_stack`/`get_vars_from_element_stack`. Aside from that, the behavior looks good.

Reply:
Does `op` in the description correspond to `f::Function` in the function handle of the description?

Reply:
Yeah, that doc string should be fixed. Good catch.